### PR TITLE
Add settings to MuchBetter

### PR DESCRIPTION
**Pull request by dara123**

### CODE REVIEW

File: openapi/components/schemas/GatewayAccountConfig/MuchBetter.yaml [hunk 1]
```diff
@@ -19,3 +19,14 @@ allOf:
         required:
           - merchantAccountId
           - apiKey
+      settings:
+        type: object
+        description: MuchBetter settings object
+        properties:
+          brandName:
```
Comment (ashkarpetin) [inline openapi/components/schemas/GatewayAccountConfig/MuchBetter.yaml:26]: Not required?

Comment (dara123) [reply openapi/components/schemas/GatewayAccountConfig/MuchBetter.yaml:26]: no. it's optional

**Review by ashkarpetin — DISMISSED** [review]

File: openapi/components/schemas/GatewayAccountConfig/MuchBetter.yaml [hunk 2]
```diff
@@ -19,3 +19,14 @@ allOf:
...
+            type: string
+            description: MuchBetter brand name use for reporting and logo purpose.
+          extraStep:
```
Comment (adamaltman) [inline openapi/components/schemas/GatewayAccountConfig/MuchBetter.yaml:29]: Boolean properties should start with `is` or `has`.

```suggestion
          hasPhoneNumberRequest:
```

File: openapi/components/schemas/GatewayAccountConfig/MuchBetter.yaml [hunk 3]
```diff
@@ -19,3 +19,14 @@ allOf:
...
+            type: boolean
+            description: Use to dispay and collect phone number before submit request to MuchBetter
```
Comment (adamaltman) [inline openapi/components/schemas/GatewayAccountConfig/MuchBetter.yaml:31]:
```suggestion
            description: Request phone number before submitting the request to MuchBetter.
```

File: openapi/components/schemas/GatewayAccountConfig/MuchBetter.yaml [hunk 4]
```diff
@@ -19,3 +19,14 @@ allOf:
         required:
           - merchantAccountId
           - apiKey
+      settings:
+        type: object
+        description: MuchBetter settings object
+        properties:
+          brandName:
+            type: string
+            description: MuchBetter brand name use for reporting and logo purpose.
```
Comment (adamaltman) [inline openapi/components/schemas/GatewayAccountConfig/MuchBetter.yaml:28]:
```suggestion
            description: MuchBetter brand name used for reporting and logo.
```

**Review by adamaltman — DISMISSED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Review by adamaltman — APPROVED** [review]

**Pull request merged**